### PR TITLE
Feature/windows terminal launcher

### PR DESCRIPTION
**Pull request by georgik**
New command which allows to register launcher for Windows terminal.
The registration is done via Windows Terminal Fragment JSON located in user's AppData.
The launcher assumes presence of ESP-IDF initializer for PowerShell.

### CODE REVIEW

File: src/launcher.rs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+fn get_add_runner(_args: &str, matches: &clap::ArgMatches<'_>) -> std::result::Result<(), clap::Error> {
+    let title = matches.value_of("title").unwrap();
+    let idf_path = matches.value_of("idf-path").unwrap();
+    let fragments_path = get_windows_terminal_fragments_path(title);
+    let tools_path = get_tools_path();
+
+    // After fresh installation of Windows Terminal the fragment path does not exist.
+    // Microsoft recommends to create one
+    // if !Path::new(&fragments_path).exists() {
```
Comment (kumekay) [inline src/launcher.rs:22]: Is it a forgotten commented out line?

File: src/launcher.rs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    fs::create_dir_all(&fragments_path)?;
+    let fragment_json_path = format!("{}/fragment.json", fragments_path);
+    println!("Updating Windows Terminal Fragment: {}", fragment_json_path);
+
+    let command_line = format!("C:/WINDOWS/System32/WindowsPowerShell/v1.0/powershell.exe -ExecutionPolicy Bypass -NoExit -File {}/Initialize-Idf.ps1", tools_path);
```
Comment (kumekay) [inline src/launcher.rs:27]: I would suggest to use `windir` env variable instead of hardcoding path on drive C:

**Review by kumekay — COMMENTED** [review]
Nice addition, thank you Juraj!

File: src/launcher.rs (hunk 2 above)
Comment (georgik) [reply src/launcher.rs]: Great idea. Thank you. Updated.

**Pull request merged**